dinstaller-cli/src/config.rs

```diff
@@ -0,0 +1,59 @@
+use clap::Subcommand;
+use std::{collections::HashMap, error::Error, str::FromStr};
+use dinstaller_lib::settings::{
+    ItemsRepository, Key as SettingsKey, Store as SettingsStore
+};
+
+#[derive(Subcommand, Debug)]
+pub enum ConfigCommands {
+    /// Set one or many installation settings
+    Set {
+        /// key-value pairs (e.g., user.name="Jane Doe")
+        values: Vec<String>,
+    },
+    /// Shows the value of one or many configuration settings
+    Show {
+        /// Keys to show
+        keys: Vec<String>,
+    },
+}
+
+pub enum ConfigAction {
+    Set(HashMap<SettingsKey, String>),
+    Show(Vec<SettingsKey>)
+}
+
+pub fn run(subcommand: ConfigCommands) -> Result<(), Box<dyn Error>> {
+    match parse_config_command(subcommand) {
+        ConfigAction::Set(changes) => {
+            let store = SettingsStore::new()?;
+            let mut model = store.load()?;
+            let settings_items = ItemsRepository::default_repository()?;
+
+            for (key, value) in &changes {
+                if let Some(item) = settings_items.find_by_key(key) {
+                    (item.update_handler)(&mut model, value)
```

Comment:
and this is new syntax for me. What it means? I kind of expect `item.update_handler(&mut model, value)`